Brokerages/InteractiveBrokers/InteractiveBrokersBrokerage.cs

```diff
@@ -125,6 +126,11 @@ public override bool IsConnected
         }
 
         /// <summary>
+        /// Returns true if the connected user is a financial advisor
+        /// </summary>
+        public bool IsFinancialAdvisor => _account.Contains("F");
```

Comment:
Contains may be sufficient, but I thought it was explicitly a starts with relationship

Reply:
We have `Fxxxxx` accounts for real FA accounts and `DFxxxxx` for paper FA accounts